exec/csnex/Support.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace csnex
+{
+    public class Support
+    {
+        static public List<string> Paths;
+
+        public Support(string source_path)
+        {
+            Paths = new List<string>();
+            // Always check current directory;
+            Paths.Add(string.Format(".{0}", Path.DirectorySeparatorChar));
+            // Also check the Neon executable path.
+            Paths.Add(string.Format("{0}{1}", source_path, Path.DirectorySeparatorChar));
+            // Then check the cwd.
+            Paths.Add(Environment.CurrentDirectory);
```

Comment:
What is the purpose of this? You have already added the current directory above.

Reply:
Removed.